src/backend/distributed/commands/function.c

```diff
@@ -1373,27 +1373,35 @@ PostprocessCreateFunctionStmt(Node *node, const char *queryString)
 		return NIL;
 	}
 
-	ObjectAddress functionAddress = GetObjectAddressFromParseTree((Node *) stmt, false);
+	List *commands = list_make1(DISABLE_DDL_PROPAGATION);
+
+	List *functionAddresses = GetObjectAddressFromParseTree((Node *) stmt, false);
+	Assert(list_length(functionAddresses) == 1);
```

Comment:
I think where we add `Assert(list_length(functionAddresses) == 1);` we should not change the code to iterate over the list